src/pymor/parameters/functionals.py

```diff
@@ -234,3 +235,90 @@ def evaluate(self, mu=None):
 
     def d_mu(self, component, index=()):
         return self.with_(constant_value=0, name=self.name + '_d_mu')
+
+
+class MinThetaParameterFunctional(ParameterFunctionalInterface):
+    """|ParameterFunctional| implementing the min-theta approach from [Haa17]_ (Proposition 2.35).
```

Comment:
Please add some more documentation (e.g. the definition of the functional) so that this can make sense without looking up the reference.